.gitlab-ci.yml

```diff
@@ -278,6 +278,26 @@ build/cuda92/intel/cuda/release/static:
     - cuda
     - gpu
 
+# Build CUDA NVIDIA without omp
+build/cuda92/intel/cuda_wo_omp/release/shared:
```

Comment:
Do we need this job ? Our CI is quite overloaded. Can we replace one of the existing job instead adding new ones ?

Reply:
We have the CCACHE now and segment ci later, so I keep these two in this pr and may change them in segment ci pr